test/unit/util/get-all-keys.js

```diff
@@ -0,0 +1,40 @@
+import keys from '../../../src/util/get-all-keys';
+import createSymbol from '../../../src/util/create-symbol';
+
+describe('utils/get-all-keys:', () => {
+  it('should return normal keys from an object', () =>
+    expect(keys({ foo: 1, bar: 2 }))
+      .to.have.lengthOf(2)
+      .and.to.deep.equal(['foo', 'bar'])
+  );
+
+  it('should return symbol keys from an object', () => {
+    const secret = createSymbol('secret');
+    expect(keys({ foo: 1, [secret]: 2 }))
+      .to.have.lengthOf(2)
+      .and.to.deep.equal(['foo', secret]);
+  });
+
+  describe('when enumerable option is set', () => {
+    const obj = {};
```

Comment:
This could just be `let obj;` and then inside the `beforeEach` block it can be

```js
obj = {};
Object.defineProperties(obj, {
  foo: { enumerable: true, value: 1 },
  bar: { enumerable: false, value: 2 },
})
```

Probably no side effects the way it is, but safer + clearer to redefine each time.